avr/scripts/progmemcheck.bat

```diff
@@ -0,0 +1,53 @@
+@echo off
+REM Windows batch file equivalent of progmemcheck.sh
+REM Checks if PROGMEM section is too large (over 65535 bytes)
+REM Usage: progmemcheck.bat <objdump_tool> <object_file>
+
+if "%1"=="" goto usage
+if "%2"=="" goto usage
+
+REM Execute objdump command and extract the address (should be decimal with -t d option)
+for /f "tokens=1" %%i in ('%1 -t d %2 ^| findstr "__ctors_start"') do set progmem_end=%%i
+
+REM Check if we got a result
+if "%progmem_end%"=="" (
+    exit /b 0
+)
+
+REM Remove any leading/trailing spaces
+set progmem_end=%progmem_end: =%
+
+REM Remove leading zeros manually (batch treats leading zeros as octal)
+:strip_zeros
+if "%progmem_end:~0,1%"=="0" if not "%progmem_end%"=="0" (
+    set progmem_end=%progmem_end:~1%
+    goto strip_zeros
+)
+
+REM If we stripped all digits, it was "000..." so set to 0
+if "%progmem_end%"=="" set progmem_end=0
+
+REM Check if progmem_end is greater than 65535
+if %progmem_end% LEQ 65535 goto end
+
+REM Calculate the excess bytes
+set /a excess=%progmem_end% - 65535
+
+REM Display warning message
+echo.
+echo _______________________________________________________________
+echo ^| Severe Warning: PROGMEM section too large by %excess% bytes.            ^|
+echo ^| Your program will most probably be unstable!                ^|
+echo ^| Use the macro PROGMEM_FAR from ^<progmem_far.h^> and          ^|
```

Comment:
"progmem_far.h" is too vague. The user won't be able to understand what is being suggested from this information. I know you provide a detailed explanation in the documentation, but the average user isn't going to read that, and it is possible to communicate this concept much more clearly without any significant increase in the verbosity of the message.

It would be better to say something like:

> Use the macro PROGMEM_FAR from the "progmem_far" library (available from Arduino Library Manager)